### PR TITLE
HW2: Cleanup proposals for Yash by rebasing HW1 on main

### PR DESCRIPTION
**Pull request by saucam**

### CODE REVIEW

File: students/yd2590.md [hunk 1]
```diff
@@ -1,47 +1,17 @@
...
-- Simple applications would not require thread safety of index structures and if they would, they could handle it on the application layer itself.
-- Take a hit (within reason) on performance to optimize memory.
-- I want to experiment with and use faster primitive data libraries like [hppc](https://labs.carrotsearch.com/hppc.html), [fastutil](https://fastutil.di.unimi.it/), [koloboke](https://github.com/leventov/Koloboke) etc for storing the vectors.
-- Experiment with storing data off-heap, to further reduce memory requirements.
-- Eventually I would like to support many different indexes since each has their own pros and cons.
-- Experiment with possibilty to use data structures that spill to disk? There is a newer [alternate hybrid algorithm](https://openreview.net/forum?id=-1rrzmJCp4) that claims to support data both in disk and in memory but its performance with different kinds of data sets is yet to be proven.
+- [shiva](./projects/scala/shiva.md)
```
Comment (timkpaine) [inline students/yd2590.md:11]: update link path and re-squash 😈

File: students/yd2590.md [hunk 2]
```diff
@@ -1,47 +1,17 @@
...
-
-Some other projects of interest:
-
-- [scikit-learn](https://github.com/scikit-learn/scikit-learn)
-- [pytorch](https://github.com/pytorch/pytorch)
-
-As I have previously worked with parquet data format and contributed some changes related  to that in apache-spark project, I think I can also contribute to the below project
-
-- [parquet4s](https://github.com/mjakubowski84/parquet4s): Parquet4s is a simple I/O for Parquet. Allows you to easily read and write Parquet files in Scala.
+- [txtai](./projects/python/txtai.md)
```
Comment (timkpaine) [inline students/yd2590.md:14]: update link paths and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/yd2590.md (hunk 1 above)
Comment (saucam) [reply students/yd2590.md]: oops, done!

File: students/yd2590.md (hunk 2 above)
Comment (saucam) [reply students/yd2590.md]: done!

**Comment by saucam** [comment]
@timkpaine thanks for the review, I have fixed the links so should be good. Maybe can take another look?

**Pull request merged**